light-client-verifier/src/operations/voting_power.rs

```diff
@@ -319,14 +329,18 @@ impl NonAbsentCommitVotes {
         };
 
         if !vote.verified {
-            let sign_bytes = vote.signed_vote.sign_bytes();
+            self.sign_bytes.truncate(0);
+            vote.signed_vote
+                .sign_bytes_into(&mut self.sign_bytes)
+                .unwrap();
```

Comment:
```suggestion
                .expect("the signing buffer should be large enough; perhaps the chain identifier is longer than 44 bytes?");
```

Reply:
The buffer is resized as needed so this only fails due to memory allocation failures.  See that [remaining_mut](https://docs.rs/bytes/1.5.0/src/bytes/buf/buf_mut.rs.html#1459) for a Vec basically returns isize::MAX and [prost checks that](https://docs.rs/prost/latest/src/prost/message.rs.html#85).